docs/en/stack/ml/df-analytics/ecommerce-outliers.asciidoc

```diff
@@ -152,12 +153,24 @@ POST _data_frame/transforms/ecommerce-customer-sales/_start
 . Create a {dfanalytics-job} to detect outliers in the new entity-centric index.
 +
 --
-There is a wizard for creating {dfanalytics-jobs} on the
-*Machine Learning* > *Data Frame Analytics* page in {kib}:
+In the wizard on the *Machine Learning* > *Data Frame Analytics* page in {kib},
+select your new index pattern then use the default values for {oldetection}. For
+example:
 
 [role="screenshot"]
 image::images/ecommerce-outlier-job-1.png["Create a {dfanalytics-job} in {kib}"]
 
+The wizard includes a scatterplot matrix, which enables you to explore the 
+relationships between the fields. You can use that information to help you
+decide which fields to include or exclude from the analysis.
+
+[role="screenshot"]
+image::images/ecommerce-outlier-scatterplot.png["A scatterplot matrix for three fields in {kib}"]
+
+If you want these charts to represent data from a larger sample size or from a
+randomized selection of documents, you can change the default behavior. However, 
+these options might slow down the performance of the matrix.
```

Comment:
> slow down the performance of the matrix

Maybe this can be clarified with a bit more detail:
- A larger sample size might slow down the performance of the matrix on the client side.
- A randomized selection will create a "heavier" query and potentially put more load on the ES cluster. It will not affect the client side performance of the matrix in comparison with a non-randomized matrix with the same sample size.

Reply:
Thanks!  I've changed the impact statement to this: "However, 
a larger sample size might slow down the performance of the matrix and a
randomized selection might put more load on the cluster due to the more
intensive query."